app/models/course.rb

```diff
@@ -869,4 +870,8 @@ def services
   def self_accredited_cannot_be_salary_funded
     errors.add(:program_type, "Salary is not valid for a self accredited course") if self_accredited? && funding_type == "salary"
   end
+
+  def accredited_body_exists_in_current_cycle
+    errors.add(:base, "The Accredited Body #{accredited_body_code} does not exist in this cycle") unless RecruitmentCycle.current.providers.find_by(provider_code: accredited_body_code)
+  end
```

Comment:
Should this return early if accredited_body_code is blank?

Reply:
Yes, added a guard clause 👍